Resources/Prototypes/_Impstation/Objectives/thief.yml

```diff
@@ -0,0 +1,453 @@
+- type: entity
+  parent: BaseThiefStealAnimalObjective
+  id: FinFinStealObjective
+  components:
+  - type: StealCondition
+    stealGroup: AnimalFinFin
+  - type: Objective
+    difficulty: 2
+
+- type: entity
+  parent: BaseThiefStealStructureObjective
+  id: GeneratorStealObjective #tesla or singulo generator
+  components:
+  - type: NotJobRequirement
+    job: StationEngineer
+  - type: StealCondition
+    stealGroup: Generator
+    descriptionText: objective-condition-thief-generator-description
+  - type: Objective
+    difficulty: 1.5
+
+- type: entity
+  parent: BaseThiefStealObjective
+  id: PunpunIDCardStealObjective
+  components:
+  - type: NotJobRequirement
+    job: Bartender
+  - type: StealCondition
+    stealGroup: PunpunID
+  - type: Objective
+    difficulty: 0.1
+
+- type: entity
+  parent: BaseThiefStealCollectionObjective
+  id: CigaretteStealCollectionObjective
+  components:
+  - type: SpeciesRequirement
+    allowedSpecies:
+      - Gray
+  - type: StealCondition
+    stealGroup: Cigarettes
+    descriptionText: objective-condition-thief-cigarettes
+    minCollectionSize: 100
+    maxCollectionSize: 1000
```

Comment:
this range should start a LOT lower and be smaller. this is pretty much "get 1x/2x 500 cigarettes and dump them all on the ground" - the 500 cigarettes item itself does not count and we really don't need a reason for people to buy it and litter the ground with entities. this should be more of a vending machine freak objective

Reply:
this objective was built around getting 500 cigarettes and leaving them at the beacon. i was not aware that the cigarettes inside 500 cigarettes dont count. going to rework this to be like 1-30 packs of cigarettes instead